gluten-ut/src/test/scala/org/apache/spark/sql/GlutenTestsTrait.scala

```diff
@@ -164,13 +171,20 @@ trait GlutenTestsTrait extends SparkFunSuite with ExpressionEvalHelper with Glut
     }
     val resultDF = df.select(Column(expression))
     val result = resultDF.collect()
-    resultDF.explain(false)
-    if (GlutenTestConstants.SUPPORTED_DATA_TYPE.contains(expression.dataType)) {
+    if (checkDataTypeSupported(expression) &&
+        !expression.children.map(checkDataTypeSupported).exists(_ == false)) {
       val projectTransformer = resultDF.queryExecution.executedPlan.collect {
         case p: ProjectExecTransformer => p
       }
-      assert(projectTransformer.size == 1)
+      if (projectTransformer.size == 1) {
+        print("Offload to native backend in the test.\n")
+      } else {
+        print("Not supported in native backend, fall back to vanilla spark in the test.\n")
+      }
```

Comment:
If only print the results about whether to support, how to check the correctness about running with Gluten?

Reply:
Thanks for your comment!
It is common that a certain UT has multiple test cases (e.g., for testing different kinds of input). Among them, some are supported by Gluten/native, but some are not. If we put that UT into black list, the supported cases in it are also excluded. 
Here, we think it is adoptable to get fallback UT executed and passed (testing the correctness of fallback code path), instead of just focusing on the UTs fully supported by Gluten/native. Right?